R/bettr-main.R

```diff
@@ -453,16 +528,15 @@ bettr <- function(df, idCol = "Method",
             for (j in values$metrics) {
                 shiny::updateNumericInput(
                     session, inputId = paste0(j, "_weight"), 
-                    value = 0.2
+                    value = initialWeightValue
```

Comment:
see comment above about promoting to main argument?

Reply:
see comment above